App.js

```diff
@@ -40,6 +44,9 @@ const port = process.env.PORT || 3000;
 //whatever is in the environment variable 'PORT', or 3000 if there's nothing there
 
 //binds and listens for connections on the specified host and port. This method is identical to Node’s http.Server.listen().
-app.listen(port);
-
-console.log(`MERN Stack listening on ${port}`);
+// app.listen(port);
+// console.log(`MERN Stack listening on ${port}`);
+http.createServer(app)
```

Comment:
use app.listen() instead of creating http server by yourself

Reply:
will fix it.